<?xml version="1.0" encoding="UTF-8"?>
<!--
Autori: Juraj Kula (xkulaj02), Michal Vaško (xvasko14)
Popis: Generovaný súbor
-->
<ui version="4.0">
 <class>mathSchemaForm</class>
 <widget class="QWidget" name="mathSchemaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="mathOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="equal">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mathSchema">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="destroySchema">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
